cmd/swarm/config.go

```diff
@@ -59,12 +59,13 @@ var (
 
 //constants for environment variables
 const (
-	SwarmEnvChequebookAddr          = "SWARM_CHEQUEBOOK_ADDR"
 	SwarmEnvAccount                 = "SWARM_ACCOUNT"
 	SwarmEnvBzzKeyHex               = "SWARM_BZZ_KEY_HEX"
 	SwarmEnvListenAddr              = "SWARM_LISTEN_ADDR"
 	SwarmEnvPort                    = "SWARM_PORT"
 	SwarmEnvNetworkID               = "SWARM_NETWORK_ID"
+	SwarmEnvChequebookAddr          = "SWARM_CHEQUEBOOK_ADDR"
+	SwarmEnvInitialDeposit          = "SWARM_INITIAL_DEPOSIT"
```

Comment:
Maybe SWARM_SWAP_INITIAL_DEPOSIT => following the convention of all SWAP parameters (but the chequebook address)

Reply:
I don't really mind, this is closely related to the chequebook and the chequebook doesn't have it either. I would vote to keep it as-is, but if you insist I will add it.

Reply:
I think it will be best to have `SWARM_SWAP_INITIAL_DEPOSIT`. Ideally, chequebook should also be in this format. I would vote for changing both, but only  `SWARM_SWAP_INITIAL_DEPOSIT` is already better than not changing it. 

It is not a blocker for me. Up to you!

Reply:
Conversation may be resolved